assets/src/edit-story/utils/test/generatePatternStyles.js

```diff
@@ -60,6 +61,10 @@ describe('generatePatternStyles', () => {
         generatePatternStyles({ color: { r: 255, g: 0, b: 0 } }, 'fill')
       ).toStrictEqual({ fill: '#f00' });
     });
+
+    it('should return an empty string for multiple values', () => {
```

Comment:
That would change this test to `'should return transparent for any non-object'`.